### DISCUSSION TITLE
Strapi v5 Support

### DISCUSSION THREAD

**Discussion by sergefy13** [Fekide/strapi-plugin-translate#480] [General]
Hey! Do you plan to migrate and support to strapi v5? Your translator is really the best on the market so it would be great to get it on the new version. Thank you!

**Comment by sargreal** [comment]
Hi @sergefy13 

we are still in progress of migrating to v5. Please follow issue #404 for updates.

**Reply by sergefy13** [reply]
Thank you for your response and for the job done!
